staging/src/k8s.io/api/admissionregistration/v1beta1/types.go

```diff
@@ -531,7 +531,8 @@ type WebhookClientConfig struct {
 	// +optional
 	Service *ServiceReference `json:"service,omitempty" protobuf:"bytes,1,opt,name=service"`
 
-	// `caBundle` is a PEM encoded CA bundle which will be used to validate the webhook's server certificate.
+	// `caBundle` is a base64 representation of a PEM-encoded CA bundle which will
```

Comment:
the base64 encoding is json/yaml-specific (proto it does not base64-encode)

Reply:
See https://github.com/kubernetes/kubernetes/pull/75199 for a (stalled) effort to document this in all relevant places. If you want to pick that up that would be great

Reply:
I see, that makes sense. I'll close this PR, as the other cover this and more.